### DISCUSSION TITLE
Websocket server should be enabled for laravel octane's swoole

### DISCUSSION THREAD

**Discussion by myckhel** [laravel/framework#43468] [Polls]
It would be a nice feature if laravel octane supports swoole's websocket.
![FYydlX3X0AI_k5X](https://user-images.githubusercontent.com/34090541/181652546-18c474ff-ac91-47b4-b36f-6e83ab1ce342.jpeg)
![FYydvplXkAA2PkP](https://user-images.githubusercontent.com/34090541/181652560-d2db95d3-1725-4713-80d5-73796f990ab6.jpeg)

**Comment by pablorsk** [comment]
Hi, do you have a temporary solution to activate laravel octane+websocket?

**Reply by myckhel** [reply]
The above screenshot 1 code works when applied in their respective files.
copy code in first screenshot into this file https://github.com/laravel/octane/blob/7ba4282697dd4f3d58704139cf02a5dfaad4e86c/bin/createSwooleServer.php#L10

copy above screenshot 2 code into this file https://github.com/laravel/octane/blob/7ba4282697dd4f3d58704139cf02a5dfaad4e86c/bin/swoole-server#L125

start the octane server and connect. it accepts normal http and ws requests

**Comment by myckhel** [comment]
Implementation: https://github.com/laravel/octane/compare/2.x...myckhel:octane:ws

Tweet: https://twitter.com/myckhelz/status/1680271083608461312?s=20

**Comment by Forsakenrox** [comment]
Nice try
